web/src/components/SearchListFilters/SearchListFilters.tsx

```diff
@@ -129,7 +129,7 @@ export const SearchListFilters: FC<SearchListFiltersProps> = ({
 			<input type="hidden" name="s" value={sortOrder} />
 			{showTextFilter ? (
 				<InlineTextFilter
-					label="Filter by title or keyword"
+					label="Keyword in title or reference number"
```

Comment:
This will need to be removed now that we don't have a label prop for `InlineTextFilter`.